packages/connect-conformance/package.json

```diff
@@ -8,12 +8,19 @@
     "conformancecloudflareserver": "bin/conformancecloudflareserver",
     "connectconformance": "bin/connectconformance"
   },
+  "main": "./dist/cjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js"
+    }
+  },
   "scripts": {
     "generate": "buf generate buf.build/connectrpc/conformance:v1.0.2",
     "clean": "rm -rf ./dist/cjs/*",
     "build": "npm run build:cjs && npm run build:esm",
-    "build:cjs": "tsc --project tsconfig.json --module commonjs --outDir ./dist/cjs",
-    "build:esm": "tsc --project tsconfig.json --module ES2015 --outDir ./dist/esm",
+    "build:cjs": "tsc --project tsconfig.json --module commonjs --outDir ./dist/cjs --declaration --declarationDir ./dist/cjs",
```

Comment:
Is it necessary to build CJS in this package?

Reply:
We do it for the binaries in the `bin` directory.

Reply:
Gotcha, I figured it was something like that. Couldn't the binaries just use `import` statements though (and have `.js` extensions)?